backend/run.sh

```diff
@@ -37,6 +38,17 @@ gonogo "migrate_app_tables"
 api_standup
 gonogo "api_standup"
 
+#####
+# MATERIALIZED VIEWS
+# These are unmanaged from Django's POV.
+# So, we don't have strictly have to drop them.
+# But, we might change them between deploys.
+# In which case, we should drop and recreate.
+drop_materialized_views
+gonogo "drop_materialized_views"
+create_materialized_views
+gonogo "create_materialized_views"
```

Comment:
We might not want to drop-and-create on every deploy. The DB is completely locked while creating the MV, which means no one can interact while the (long-running) create command is running. We should consider building once, then running a **concurrent** refresh on deploy and on a schedule. There will still be some downtime during the refresh, but it _should_ be substantially shorter.

Reply:
I hadn't gotten that far, but having something in here that checks to see if it exists, and then *does nothing* on deploy, is possible.

I have an example, in `bash`, of this over in... the code that loads the historic data.